### PR TITLE
Add 2022 Keynote Nomination Post

### PR DESCRIPTION
**Pull request by nhandler**
This change adds a new blog post announcing the 2022 Keynote
Nominations.

### CODE REVIEW

File: _posts/2022-04-05-keynote-nomination.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+status: publish
+type: post
+published: true
+categories: news
+tags: '2022'
+---
+
+One of the things that makes SeaGL unique from other conferences is that we allow the public to play a role in our keynote selection process. For the next two weeks, we will be accepting nominations for people you would like to see present this year's keynote talks.
+
+While you are welcome to self-nominate yourself, we encourage you to take some time to think about other people you have seen present in the past who you think have an important message to share and nominate them as well.
```
Comment (funnelfiasco) [inline _posts/2022-04-05-keynote-nomination.md:13]: "self-nominate yourself" may be slightly redundant. :-)

File: _posts/2022-04-05-keynote-nomination.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+layout: post
+title: 'Now Accepting Nominations for SeaGL 2022 Keynotes'
+status: publish
+type: post
+published: true
+categories: news
+tags: '2022'
+---
+
+One of the things that makes SeaGL unique from other conferences is that we allow the public to play a role in our keynote selection process. For the next two weeks, we will be accepting nominations for people you would like to see present this year's keynote talks.
```
Comment (funnelfiasco) [inline _posts/2022-04-05-keynote-nomination.md:11]: "we allow the public to play a role" sounds a little...IDK. Words. Maybe instead something like "our community helps us select our keynote speakers." Focus more on participation and not permission, if that makes sense.

File: _posts/2022-04-05-keynote-nomination.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+We challenge you to think about people who
+  * You go out of your way to read all of their Open Source posts on social media
+  * Might only have a small following, but are doing amazing work in their own little corner
+  * Has done work to empower others and challenge social norms
+  * Are always tinkering with new and exciting projects that inspire others to try new things
+  * Are slowly, but steadily, working to make the Open Source community and the world a better place for everyone
+and [nominate them](https://docs.google.com/forms/d/e/1FAIpQLSfvBo7SX6ku6vmLLvtzbJ_c80b_WfvhUl-GAXpc6nssXCvg-w/viewform?usp=sf_link) to be a SeaGL Keynote presenter!
+
+SeaGL will take place virtually November 4-5, 2022.
```
Comment (funnelfiasco) [inline _posts/2022-04-05-keynote-nomination.md:27]: We are discussing the format as I type this.

File: _posts/2022-04-05-keynote-nomination.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+One of the things that makes SeaGL unique from other conferences is that we allow the public to play a role in our keynote selection process. For the next two weeks, we will be accepting nominations for people you would like to see present this year's keynote talks.
+
+While you are welcome to self-nominate yourself, we encourage you to take some time to think about other people you have seen present in the past who you think have an important message to share and nominate them as well.
+
+To nominate someone, please use our [keynote speaker nomination form](https://docs.google.com/forms/d/e/1FAIpQLSfvBo7SX6ku6vmLLvtzbJ_c80b_WfvhUl-GAXpc6nssXCvg-w/viewform?usp=sf_link). You can submit as many nominations as you want (the more the merrier!), and all submissions are copmletely anonymous. **The form will be open until Friday, April 22 at 11:59pm PDT.**
+
+Nominees need not have prior keynote experience; we are interested in both new and veteran keynote presenters. If they have been doing exciting work in the Open Source community and have and exciting story to share, we want to know about them! We also strive to have representation from a broad range of backgrounds.
```
Comment (funnelfiasco) [inline _posts/2022-04-05-keynote-nomination.md:17]: I know we don't have a conference theme yet, but is there a theme or a...general sort of thing we'd like to publicly say we're focusing on. ("No" is a fine answer, but if we do, we should call it out)

**Review by funnelfiasco — CHANGES_REQUESTED** [review]

File: _posts/2022-04-05-keynote-nomination.md (hunk 4 above)
Comment (altsalt) [reply _posts/2022-04-05-keynote-nomination.md:17]: Yeah ten... but nothing has quite felt right yet.

File: _posts/2022-04-05-keynote-nomination.md (hunk 1 above)
Comment (nhandler) [reply _posts/2022-04-05-keynote-nomination.md:13]:
```suggestion
While you are welcome to nominate yourself, we encourage you to take some time to think about other people you have seen present in the past who you think have an important message to share and nominate them as well.
```

File: _posts/2022-04-05-keynote-nomination.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+published: true
+categories: news
+tags: '2022'
+---
+
+One of the things that makes SeaGL unique from other conferences is that we allow the public to play a role in our keynote selection process. For the next two weeks, we will be accepting nominations for people you would like to see present this year's keynote talks.
+
+While you are welcome to self-nominate yourself, we encourage you to take some time to think about other people you have seen present in the past who you think have an important message to share and nominate them as well.
+
+To nominate someone, please use our [keynote speaker nomination form](https://docs.google.com/forms/d/e/1FAIpQLSfvBo7SX6ku6vmLLvtzbJ_c80b_WfvhUl-GAXpc6nssXCvg-w/viewform?usp=sf_link). You can submit as many nominations as you want (the more the merrier!), and all submissions are copmletely anonymous. **The form will be open until Friday, April 22 at 11:59pm PDT.**
```
Comment (nhandler) [inline _posts/2022-04-05-keynote-nomination.md:15]:
```suggestion
To nominate someone, please use our [keynote speaker nomination form](https://docs.google.com/forms/d/e/1FAIpQLSfvBo7SX6ku6vmLLvtzbJ_c80b_WfvhUl-GAXpc6nssXCvg-w/viewform?usp=sf_link). You can submit as many nominations as you want (the more the merrier!), and all submissions are completely anonymous. **The form will be open until Friday, April 28 at 11:59pm PDT.**
```

File: _posts/2022-04-05-keynote-nomination.md (hunk 3 above)
Comment (nhandler) [reply _posts/2022-04-05-keynote-nomination.md:27]:
```suggestion
SeaGL will take place November 4-5, 2022.
```

**Comment by funnelfiasco** [comment]
Shared on Facebook, Twitter, Mastodon, and LinkedIn. I'll share on Twitter/Mastodon several more times and then on all four again on the 28th

**Pull request merged**